feat(execute-driver-plugin): use peer deps

boneskull opened the pull request:
BREAKING CHANGE:

`@appium/execute-driver-plugin` now expects to be installed alongside `appium`.

boneskull commented:
Current dependencies on/for this PR:
* 2.0
  * **PR #16901** <a href="https://app.graphite.dev/github/pr/appium/appium/16901" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a> 
    * **PR #16902** <a href="https://app.graphite.dev/github/pr/appium/appium/16902" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a>  (this PR)
    * **PR #16903** <a href="https://app.graphite.dev/github/pr/appium/appium/16903" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a> 
    * **PR #16904** <a href="https://app.graphite.dev/github/pr/appium/appium/16904" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a> 
    * **PR #16905** <a href="https://app.graphite.dev/github/pr/appium/appium/16905" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a> 
    * **PR #16906** <a href="https://app.graphite.dev/github/pr/appium/appium/16906" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a> 
    * **PR #16907** <a href="https://app.graphite.dev/github/pr/appium/appium/16907" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a> 
    * **PR #16908** <a href="https://app.graphite.dev/github/pr/appium/appium/16908" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/appium/appium/16902?utm_source=stack-comment).

mykola-mokhnach commented on packages/execute-driver-plugin/test/e2e/plugin.e2e.spec.js:
```diff
@@ -2,30 +2,38 @@
-import { remote as wdio } from 'webdriverio';
-import { W3C_ELEMENT_KEY, MJSONWP_ELEMENT_KEY } from '../../lib/execute-child';
-import { fs } from '@appium/support';
+import {e2eSetup as _e2eSetup} from 'appium/plugin';
```
was this import moved to the package index?

boneskull replied:
no, this is wrong. it will be moved to test-support later

mykola-mokhnach commented:
Should tests pass first?

boneskull committed:
feat(execute-driver-plugin): use peer deps

BREAKING CHANGE:

`@appium/execute-driver-plugin` now expects to be installed alongside `appium`.

# Conflicts:
#	packages/execute-driver-plugin/lib/execute-child.js
#	packages/execute-driver-plugin/lib/plugin.js
#	packages/execute-driver-plugin/test/e2e/plugin.e2e.spec.js

# Conflicts:
#	packages/execute-driver-plugin/test/e2e/plugin.e2e.spec.js

boneskull commented:
I need to add some more E2E tests around how these are handled before I'm comfortable merging them.
